src/Dfe.PlanTech.Application/Users/Helper/UserHelper.cs

```diff
@@ -41,18 +41,18 @@ public UserHelper(IHttpContextAccessor httpContextAccessor, IPlanTechDbContext d
             var reference = establishmentDto.Urn ?? establishmentDto.Ukprn;
 
             if (reference is null)
-                throw new InvalidOperationException("Establishment reference cannot be null");
+                return 1;
 
             var existingEstablishmentId = await getEstablishmentIdQuery.GetEstablishmentId(reference);
 
             if (existingEstablishmentId == null)
             {
                 await SetEstablishment();
                 var newEstablishmentId = await getEstablishmentIdQuery.GetEstablishmentId(reference);
-                return newEstablishmentId;
+                return newEstablishmentId is null ? 1 : Convert.ToUInt16(newEstablishmentId);
```

Comment:
and here?

Reply:
Noted and will be fixed as part of Jim's work.